src/dc_context.c

```diff
@@ -307,6 +336,45 @@ char* dc_get_blobdir(const dc_context_t* context)
  * INI-handling, Information
  ******************************************************************************/
 
+static int is_settable_config_key(const char* key)
+{
+	for (int i = 0; i < str_array_len(config_keys); i++) {
+		if (strcmp(key, config_keys[i]) == 0) {
+			return 1;
+		}
+	}
+	return 0;
+}
+
+
+static int is_gettable_config_key(const char* key)
+{
+	for (int i = 0; i < str_array_len(sys_config_keys); i++) {
+		if (strcmp(key, sys_config_keys[i]) == 0) {
+			return 1;
+		}
+	}
+	return is_settable_config_key(key);
+}
+
+
+static char* get_config_keys_str() {
```

Comment:
just nitpicking:
linebreak missing (there is no concrete style guide, however, typically we use linebreaks after function names to have a clearer separation from for/while/if